source/patterns/@aws-solutions-constructs/aws-eventbridge-lambda/lib/index.ts

```diff
@@ -45,6 +57,7 @@ export interface EventbridgeToLambdaProps {
 
 export class EventbridgeToLambda extends Construct {
   public readonly lambdaFunction: lambda.Function;
+  public readonly eventBus?: events.IEventBus;
```

Comment:
`default` EventBus is always there per AWS region, the construct and the CDK considers `default` EventBus if `IEventBus` is not explicitly provided by the user. In case of CDK, `events.RuleProps.eventBus` is an optional parameter. We could always return an IEventBus by explicitly looking up the `default` EventBus if user of the construct is not supplying it. Open for suggestions.

Reply:
These will probably usually be deployed in pairs, such as:
```
const ev = new aws-eventbridge-lambda(this, 'id', {
  eventBusProps: {}
});

aws-lambda-eventbridge(this, 'id2', {
  existingEventBus: ev.eventBus,
});
```

My thinking is that this logic is the same whether the event bus was custom or the default. I don't want customers to have to write code like:

```
if (ev.EventBus) {
  aws-lambda-eventbridge(this, 'id2', {
    existingEventBus: ev.eventBus,
  });
} else {
  aws-lambda-eventbridge(this, 'id2', {});
}
```

It's not a huge deal - if the default bus can't be referenced in an eventBus construct like custom buses then so be it.

Reply:
I share your concern, we could still reference default event bus explicitly using one of the static methods available `events.EventBus.fromXxxxx()`.  But while creating certain Rules, especially `schedule` rule, CDK doesn't allow providing `eventBus` and `schedule` as part of `event.RuleProps`. Please refer line 127 of [Rule.ts](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-events/lib/rule.ts) implementation

Reply:
Got it - OK.